### DISCUSSION TITLE
Ability to Change Label Text on checkout or Basket page when using Wordpress/woocommerce blocks

### DISCUSSION THREAD

**Discussion by turnkeymaker** [woocommerce/woocommerce#49738] [Checkout Flow]
One of the label text on the checkout appears as "Taxes". In the UK, this is meant to be described as "VAT" , so it would be ideal for the label to be editable so that the text can be modified accordingly.
![image](https://github.com/user-attachments/assets/6a0626e3-fea6-4c07-8511-b4295b910fe0)

**Comment by senadir** [comment]
Hey! Name your taxes in your admin screen

<img width="1063" alt="image" src="https://github.com/user-attachments/assets/8560b54e-faa1-44c1-be5e-dc32e078e8c5">


<img width="369" alt="image" src="https://github.com/user-attachments/assets/f6ed37be-a0a8-403c-abac-750bea4b0d4c">

For it to show up as a single line item, you need to show taxes excluded in Checkout and itemized

<img width="677" alt="image" src="https://github.com/user-attachments/assets/77cd658b-e8fd-493d-a793-3bf4ef9b7823">

**Comment by turnkeymaker** [comment]
Hello,
    Thanks for your reply.

It's already named as VAT in the txt section of the admin, but still
remains as Tax on checkout page.


Regards Fred

**Reply by senadir** [reply]
Make sure your settings matches the one above.

**Comment by turnkeymaker** [comment]
Hello,
      looks like the itemised tax option setting was what didn't allow the
VAT to appear.
Thanks

Regards Fred
